pkg/nodeconfig/nodeconfig.go

```diff
@@ -66,6 +66,38 @@ type nodeConfig struct {
 	additionalLabels map[string]string
 }
 
+// ErrWriter is a wrapper to enable error-level logging inside kubectl drainer implementation
```

Comment:
Won't a commit message `[nodeconfig] Rearrange ErrWriter` rather than a generic refactoring message?

Reply:
Again, `[nodeconfig] Re-arrange constructor and structs` is too generic. Think about someone reading the git log without checking the git messages itself.

Reply:
Right, re-worded in https://github.com/openshift/windows-machine-config-operator/pull/676/commits/bd795161d5bb057b7e27edfe429df25f3be8cc53

Reply:
Sorry to be so pedantic ` [nodeconfig] Re-arrange newDrainHelper()` is not correct either. How about  `[nodeconfig] Move *Writer and associated methods`? 

>Move the drainerHelper constructor and OutWriter, ErrWriter struct
definitions to the beginning of the file, easier to find and maintain
as the file grows.

What is drainerHelper? Please fix.